ssv/aggregator.go

```diff
@@ -29,7 +29,12 @@ func NewAggregatorRunner(
 	operatorSigner *types.OperatorSigner,
 	valCheck qbft.ProposedValueCheckF,
 	highestDecidedSlot phase0.Slot,
-) Runner {
+) (Runner, error) {
+
+	if len(share) != 1 {
+		return nil, errors.New("must have only one share")
```

Comment:
if we add an error we should have an accompaning test

Reply:
On it